packages/unmock-core/src/service/serviceStore.ts

```diff
@@ -55,12 +65,12 @@ export class ServiceStore {
       // i.e. `state.github.get(...).post(...)` // make sure both `get` and `post` are correct methods
       throw new Error(
         `Can't find specification for service named '${
-          isExtendedRESTMethod(method) ? service : method
+          isExtendedRESTMethod(method) ? serviceName : method
         }'!`,
       );
     }
 
-    const error = this.serviceMapping[service].updateState({
+    const error = this.serviceMapping[serviceName].updateState({
```

Comment:
Is this still returning errors or throwing? 🤔

Reply:
Woops! Nice catch.